static/explorer.scss

```diff
@@ -247,6 +247,16 @@ pre.content.wrap * {
     color: red !important;
 }
 
+.flow-decoration {
+    background-color: #b4131f !important;
+    color: white !important;
+    font-weight: bold;
+}
+
+.flow-highlight {
+    background-color: rgba(180, 19, 31, 0.4) !important;
```

Comment:
We prefer to define colours in the `static/theme/*-theme.scss` files, so that we can properly change them for each variant

Reply:
moved